### PR TITLE
OCPBUGS-19346: set default deploymentconfig params on AWS CCM

### PR DESCRIPTION
**Pull request by sjenning**
https://issues.redhat.com/browse/OCPBUGS-19346

cc @csrwng

### CODE REVIEW

File: control-plane-operator/controllers/hostedcontrolplane/cloud/aws/params.go [hunk 1]
```diff
@@ -31,5 +35,21 @@ func NewAWSParams(hcp *hyperv1.HostedControlPlane) *AWSParams {
 	}
 	p.OwnerRef = config.ControllerOwnerRef(hcp)
 
+	p.DeploymentConfig.SetDefaults(hcp, ccmLabels(), pointer.Int(1))
```
Comment (sjenning) [inline control-plane-operator/controllers/hostedcontrolplane/cloud/aws/params.go:38]: This is the main difference.  The rest is a refactor to make it more like the param/reconcile pattern in the other components.

File: control-plane-operator/controllers/hostedcontrolplane/cloud/aws/params.go [hunk 2]
```diff
@@ -31,5 +35,21 @@ func NewAWSParams(hcp *hyperv1.HostedControlPlane) *AWSParams {
...
+		ccmContainer().Name: {
+			Requests: corev1.ResourceList{
+				corev1.ResourceMemory: resource.MustParse("60Mi"),
+				corev1.ResourceCPU:    resource.MustParse("75m"),
+			},
+		},
+	}
+	p.DeploymentConfig.AdditionalLabels = additionalLabels()
+	p.DeploymentConfig.Scheduling.PriorityClass = config.DefaultPriorityClass
+	if hcp.Annotations[hyperv1.ControlPlanePriorityClass] != "" {
```
Comment (sjenning) [inline control-plane-operator/controllers/hostedcontrolplane/cloud/aws/params.go:49]: This logic was also missing.

**Comment by csrwng** [comment]
Seems like an issue with CI registry:
```
Failed to pull image "registry.build02.ci.openshift.org/ci-op-x3vppmgy/pipeline@sha256:9154f95ec19804bb07d146f105cf329bf8a613e1bff631bd9d1057f4b099e796": rpc error: code = DeadlineExceeded desc = pinging container registry registry.build02.ci.openshift.org: Get "https://registry.build02.ci.openshift.org/v2/": dial tcp 34.74.144.21:443: i/o timeout
```

/retest-required

**Pull request merged**